man/plot_layout.Rd

```diff
@@ -35,7 +35,8 @@ Returns a plotly object which contains the layout
   that is used for probability plotting.
```

Comment:
Could you please check the doc of plot_layout? I think "returns a plotly object" is written there...